corehq/apps/geospatial/static/geospatial/js/geospatial_map.js

```diff
@@ -419,7 +419,15 @@ hqDefine("geospatial/js/geospatial_map", [
         // Hide the datatable rows but not the pagination bar
         $('.dataTables_scroll').hide();
 
-        if (xhr.responseJSON.aaData.length && mapModel.mapInstance) {
+        if (xhr.status !== 200) {
+            if (xhr.responseText.length) {
+                alertUser.alert_user(xhr.responseText, 'danger');
+            } else {
+                alertUser.alertUser(
```

Comment:
This is camel case here, and snake case on line 424. Is that correct?

Reply:
Good catch. Looks like this should be snake case

Reply:
Ah yes, this should indeed be snake case. Addressed in [fe127d4](https://github.com/dimagi/commcare-hq/pull/34895/commits/fe127d4ed96f3d7485f6f02bee533e45e61cc71a).